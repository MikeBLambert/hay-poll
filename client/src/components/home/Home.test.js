import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';

describe('Home component', () => {
  it('renders a home component', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
