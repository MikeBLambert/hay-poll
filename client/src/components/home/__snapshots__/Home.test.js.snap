// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders a home component 1`] = `
<div>
  <h1>
    Hello Home!
  </h1>
</div>
`;
